ensawards.org/src/contract-pipelines/filters.ts

```diff
@@ -0,0 +1,35 @@
+import type { Contract, ContractType } from "@/types/contracts.ts";
+import { ContractTypes } from "@/types/contracts.ts";
+
+/**
+ * Filters contracts by their type field.
+ *
+ * This is useful for creating category-specific leaderboards where we want to ensure
+ * we're comparing apples to apples. For example, on the DAO leaderboard, we only want
+ * to include contracts that are DAO-related (governance, treasury, etc.) and exclude
+ * DeFi protocol contracts, even if they belong to a DAO organization.
```

Comment:
```suggestion
 * to include contracts that are DAO-related (governance, treasury, etc.) and exclude
 * DeFi protocol contracts, even if they belong to the same project the DAO is associated with.
```

Goal: Align with the taxonomy we're building where project is at the top.